### PR TITLE
*: unified region split message

### PR DESCRIPTION
**Pull request by overvenus**
Remove `SplitCheckResult` for unifying region split messages.

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

PTAL @BusyJay @javaforfun

File: src/raftstore/store/worker/split_check.rs [hunk 1]
```diff
@@ -249,11 +249,13 @@ impl<C: Sender<Msg>> Runnable<Task> for Runner<C> {
...
-    Msg::SplitCheckResult {
+fn new_split_region(region_id: u64, epoch: RegionEpoch, split_key: Vec<u8>) -> Msg {
+    let key = keys::origin_key(split_key.as_slice()).to_vec();
+    Msg::SplitRegion {
         region_id: region_id,
-        epoch: epoch,
-        split_key: split_key,
+        region_epoch: epoch,
+        split_key: key,
+        callback: Box::new(|_| {}),
```
Comment (BusyJay) [inline src/raftstore/store/worker/split_check.rs]: Can it be optional?

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**